network-api/networkapi/wagtailpages/tests/buyersguide/test_product_page_evaluation.py

```diff
@@ -192,10 +192,105 @@ def test_creepiness_per_bin_limits(self):
         self.assertEqual(evaluation.average_creepiness, 49.5)
         self.assertEqual(evaluation.total_creepiness, 495)
         self.assertEqual(evaluation.total_votes, 10)
+        self.assertDictEqual(
+            evaluation.average_bin, {"label": "Somewhat creepy", "localized": gettext("Somewhat creepy")}
+        )
+
+
+class TestProductPageEvaluationAverageBin(BuyersGuideTestCase):
+    def setUp(self):
+        super().setUp()
+        self.admin_user = self.create_superuser(username="admin", password="password")
+        self.login(self.admin_user)
+        self.product_page = buyersguide_factories.ProductPageFactory(parent=self.bg)
+
+    def test_avg_bin_with_no_votes(self):
+        evaluation = (
+            ProductPageEvaluation.objects.with_total_votes()
+            .with_total_creepiness()
+            .with_average_creepiness()
+            .get(pk=self.product_page.evaluation.pk)
+        )
+
+        self.assertEqual(evaluation.total_votes, 0)
+        self.assertDictEqual(evaluation.average_bin, {"label": "No votes", "localized": gettext("No votes")})
+
+    def test_avg_bin_with_avg_vote_equal_0(self):
+        buyersguide_factories.ProductVoteFactory(evaluation=self.product_page.evaluation, value=0)
+        evaluation = (
+            ProductPageEvaluation.objects.with_total_votes()
+            .with_total_creepiness()
+            .with_average_creepiness()
+            .get(pk=self.product_page.evaluation.pk)
+        )
+
+        self.assertEqual(evaluation.average_creepiness, 0)
+        self.assertDictEqual(evaluation.average_bin, {"label": "Not creepy", "localized": gettext("Not creepy")})
+
+    def test_avg_bin_with_avg_vote_between_0_and_20(self):
```

Comment:
For these tests in general, should we be creating multiple votes (e.g., `0` and `20`) instead of just one with the average creepiness (e.g., `10`) so that we can test out `with_average_creepiness()` more?

Reply:
I've changed the tests to work with random ints on the bin's range

Reply:
```suggestion
    def test_avg_bin_with_avg_vote_between_1_and_20(self):
```

to follow the naming convention of the rest of the tests

Reply:
Done!